config/locales/client.en.yml

```
@@ -7308,16 +7315,17 @@ en:
 
           <p>Replace <b>DISCOURSE_USERNAME</b> with the Discourse username of the author that should create the topic. Discourse will automatically lookup the user by the <code>content</code> attribute of the <code>&lt;meta&gt;</code> tags with <code>name</code> attribute set to <code>discourse-username</code> or <code>author</code>. The <code>discourseUserName</code> parameter has been deprecated and will be removed in Discourse 3.2.</p>
         title: "Embedding"
+        description: "Discourse has the ability to embed the comments from a topic in a remote site using a Javascript API that creates an IFRAME"
         host: "Allowed Hosts"
         allowed_paths: "Path Allowlist"
         edit: "edit"
         category: "Post to Category"
         tags: "Topic Tags"
         post_author: "Post Author - Defaults to %{author}"
         add_host: "Add Host"
```

```suggestion
        add_host: "Add host"
```

While you are here ;)